src/components/ProfileDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import CommunityGrid from './CommunityGrid';
+import StudentGrid from './StudentGrid';
+
+export default function ProfileDisplay() {
+  const [user, setUser] = React.useState(0);
+
+  const handleChange = (event, newUser) => {
+    setUser(newUser);
+  };
+
+  function displaySelection(userSelected) {
+    if (userSelected === 0) {
+      return <StudentGrid />
+    }
+    return <CommunityGrid />
+  }
+
+
+  return (
+    <div className="stepperStyle">
+      <Paper square>
+        <Tabs value={user} indicatorColor="primary" textColor="primary" onChange={handleChange}>
+          <Tab label="Student" />
+          <Tab label="Community Contributor" />
+        </Tabs>
+        {displaySelection(user)}
```

Comment:
No need to conditionally display one or the other. Use a [`<TabPanel>`](https://material-ui.com/components/tabs/) here to hold each component.

Reply:
Ah ok I see then this will fix the comment above about the `user` having a value 0

Reply:
<TabPanel> is fully functional at the moment, thanks for that link!